contributing/README.md

```diff
@@ -104,20 +104,114 @@ Signed-off-by: Random J Developer <random@developer.example.org>
 Git even has a -s command line option to append this automatically to your
 commit message:
 
-```
-$ git commit -s -m 'This is my commit message'
+```bash
+git commit -s -m 'This is my commit message'
 ```
 
 If you have already made a commit and forgot to include the sign-off, you can
 amend your last commit to add the sign-off with the following command, which can
 then be force pushed.
 
-```
+```bash
 git commit --amend -s
 ```
 
 We use a [DCO bot] to enforce the DCO on all commits in every pull request.
 
+## Code Review Process
+
+All Pull Requests (PR), whether written by a Crossplane maintainer or a
+community member, must go through code review.
+
+Not only do code reviews ensure that the code is correct, maintainable, and
+secure, but more importantly it allows us to use code reviews as an educational
+tool.
+
+With this in mind, all efforts around code reviews should be seen through the
+lens of educating the reviewee and be accompanied by kind, detailed feedback
```

Comment:
i got confused here by "reviewee" and read it as "reviewer" - what do you think about using terminology of "contributor"/"author" (the person opening the PR) and "reviewer" (the person reviewing the PR)? 🤔 

Looks like we are using "author" in the section header below, so maybe standardizing on that would be good?

Reply:
Yip, I agree with being consistent. I'm updating it to "author".